config/crd/bases/api-gateway.consul.hashicorp.com_gatewayclassconfigs.yaml

```diff
@@ -59,12 +59,12 @@ spec:
                     description: Consul authentication information
                     properties:
                       account:
-                        description: The Kubernetes service account to authenticate
-                          as.
+                        description: The name of an existing Kubernetes ServiceAccount
+                          to authenticate as. Ignored if !managed.
```

Comment:
```suggestion
                          to authenticate as. Ignored if managed is true.
```

Reply:
integrated your feedback in `types.go` where these CRDs are generated from

Reply:
We use the `controller-gen` tool by running `$ make generate`